src/content/docs/en/concepts/islands.mdx

```diff
@@ -40,7 +40,7 @@ import MyReactComponent from '../components/MyReactComponent.jsx';
 <MyReactComponent />
 ```
 
-But sometimes, client-side JavaScript is required for creating interactive UI. Instead of forcing your entire page to become an SPA-like JavaScript application, Astro asks you to create an island.
+But sometimes, client-side JavaScript is required for creating interactive UI. Instead of forcing your entire page to become a SPA-like JavaScript application, Astro asks you to create an island.
```

Comment:
```suggestion
But sometimes, client-side JavaScript is required for creating interactive UI. Instead of forcing your entire page to become a single-page JavaScript application (SPA), Astro asks you to create an island.
```